index.html

```diff
@@ -244,9 +269,71 @@
         </dt>
       </dl>
       <p>
-        The <code>PermissionName</code> enum defines the list of known
-        permission names.
+        Each enumerator-value in the <a>PermissionName</a> enum identifies a
+        <dfn data-export="">permission</dfn>, which consists of the following
+        algorithms and types:
       </p>
+      <dl>
+        <dt>
+          An <dfn data-export="">associated PermissionDescriptor</dfn>
+        </dt>
+        <dd>
+          A subtype of <a>PermissionDescriptor</a>.
```

Comment:
Hm, I didn't check the examples before writing this, but I see just:
```javascript
{name:'geolocation'}
{name:'notifications'}
{name:'camera', deviceId: devices[0].deviceId}
```
which are all instances of subtypes of  `PermissionDescriptor`:
```webidl
dictionary PermissionDescriptor {
    required PermissionName name;
};
```

What example are you looking at?